src/main/java/com/example/auction/kafka/producer/ReactiveProducerService.java

```diff
@@ -0,0 +1,24 @@
+package com.example.auction.kafka.producer;
+
+import com.example.auction.kafka.dto.UpdateAccountCommand;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.kafka.core.reactive.ReactiveKafkaProducerTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ReactiveProducerService {
```

Comment:
@amerkul 
ReactiveKafkaConsumer and ReactiveProducerService are not unified and persistent in the naming way.
`ReactiveKafkaConsumer` & `ReactiveKafkaProducer`
OR
`ReactiveConsumerService` & `ReactiveProducerService`